webpack/assets/javascripts/react_app/components/statistics/ChartBox.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Panel } from 'react-bootstrap';
-
+import { Panel, Modal } from 'react-bootstrap';
+import Immutable from 'seamless-immutable';
 import helpers from '../../common/helpers';
-import PieChart from '../common/charts/PieChart/';
-import { getLargePieChartConfig, navigateToSearch } from '../../../services/ChartService';
-import ChartModal from './ChartModal';
+import ForemanDonutChart from '../common/charts/DonutChart';
```

Comment:
I'm not sure, why should we add foreman prefix to the file name, if its under foreman dir anyway ?

Reply:
I am mostly afraid of confusion because the component name doesn't fit with the component file/directory.

maybe it should simply be called `DonutChart`?